hq_superset/models.py

```diff
@@ -32,3 +45,71 @@ def update_dataset(self):
                 rows = list(cast_data_for_table(self.data, table))
                 insert_stmt = table.insert().values(rows)
                 connection.execute(insert_stmt)
+
+
+class HQClient(db.Model, OAuth2ClientMixin):
```

Comment:
I'm going to be nitpicking here...

Can we maybe call this something like `HQOAuthClient`? The reason is that the current naming made me think this was the request client (don't know if I'm just super pedantic).

Anyway, I'll leave it to your own discretion...

Reply:
When I rebased (before reading this comment) I renamed this class "OAuth2Client", in line with its mixin. I did the same with "Token".

"HQOAuthClient" may have been a better name, considering it's got a "domain" field, but I feel like "OAuth2Client" is close enough, and better than "CommCareHQOAuth2Client".

Reply:
> I feel like "OAuth2Client" is close enough

Perfect!